2082 document featfrontend change streaming rate limit=0 to pause stream instead of disable rate limit (#2111)
risingwavelabs/risingwave-docs: docs/manage/view-configure-runtime-parameters.md

| lock_timeout                          | 0               | See [here](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-LOCK-TIMEOUT) for details. Unused in RisingWave, support for compatibility.                                                                                                                                                            |
| row_security                          | `true`/`false`            | See [here](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-ROW-SECURITY) for details. Unused in RisingWave, support for compatibility.                                                                                                                                                          |
| standard_conforming_strings           | on              | See [here](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-STANDARD-CONFORMING-STRINGS)  for details.                                                                                                                                                                                              |

> | streaming_rate_limit                  | `0`/`default`    | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> Setting it to `0` now results in pausing the snapshot read stream for backfill and pausing the source read for sources (Previously, this disabled the rate limit within the session). Setting it to `default` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |

Reviewer: ```suggestion
| streaming_rate_limit                  |  `default`  | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> Setting it to `0` now results in pausing the snapshot read stream for backfill and pausing the source read for sources (Previously, this disabled the rate limit within the session). Setting it to `default` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |
```

Reviewer: Default value is to disable rate limit.
`default` actually maps to `-1` internally. However, if users run: `set streaming_rate_limit=-1`, our parser may actually reject it due to some limitations. So we just let users set it back to `default` for this.

Author: Thanks for the detailed explanation! Actually I'm a bit confused about why we should remove `0` from the "values" column. Does that mean we don't recommend this value? Do I need to add some additional notes?
<img width="178" alt="image" src="https://github.com/risingwavelabs/risingwave-docs/assets/150168166/89bdf0ef-149e-45fe-b8e3-af35754d4d55">

Reviewer: Oh I see. Sorry I thought the column is for default values only.

In that case this can be `default` or `int`.

> | streaming_rate_limit                  | `0`/`default`    | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> Setting it to `0` now results in pausing the snapshot read stream for backfill and pausing the source read for sources (Previously, this disabled the rate limit within the session). Setting it to `default` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |

Reviewer 2: ```suggestion
| streaming_rate_limit                  | `0`/`default`    | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> Setting it to `0` results in pausing the snapshot read stream for backfill and pausing the source read for sources (Previously, this disabled the rate limit within the session). Setting it to `default` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |
```

> | streaming_rate_limit                  |  `default`/ A positive integer / `0`  | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> `SET STREAMING_RATE_LIMIT TO 0` results in pausing the snapshot read stream for backfill and pausing the source read for sources (Previously, this disabled the rate limit within the session). `SET STREAMING_RATE_LIMIT TO DEFAULT` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |

Reviewer 3: ```suggestion
| streaming_rate_limit                  |  `default`/ A positive integer / `0`  | Set the maximum number of records per second per source, for each parallelism. The source here refers to an upstream source or snapshot read in the backfilling process.<br/><br/> `SET STREAMING_RATE_LIMIT TO 0` will pause the snapshot read stream for backfill and pause the source read for sources (Previously, this disabled the rate limit within the session). `SET STREAMING_RATE_LIMIT TO DEFAULT` will disable the rate limit within the session, but it will not change the rate limits of existing DDLs.                                                                                                                                              |
```

Reviewer: Looks good to me. Thanks!
